src/components/views/elements/Tooltip.tsx

```diff
@@ -98,6 +98,8 @@ export default class Tooltip extends React.Component<ITooltipProps> {
         });
     }
 
+    // Add the parent's position to the tooltips, so it's correctly
+    // positioned, also taking into account any window zoom
```

Comment:
Moved this comment up to describe the function instead of in the usage where the context can get lost when you're just focusing on the `updatePosition` code